src/main/kotlin/org/rust/ide/console/RsConsoleFolder.kt

```diff
@@ -0,0 +1,44 @@
+package org.rust.ide.console;
+
+import com.intellij.execution.ConsoleFolding
+import com.intellij.openapi.project.Project
+import org.rust.cargo.project.model.cargoProjects
+import org.rust.cargo.project.workspace.PackageOrigin
+import org.rust.cargo.runconfig.filters.RegexpFileLinkFilter
+import org.rust.cargo.runconfig.filters.RsBacktraceFilter
+import org.rust.cargo.runconfig.filters.RsBacktraceItemFilter
+import org.rust.lang.core.resolve.resolveStringPath
+import java.io.File
+
+class RsConsoleFolder : ConsoleFolding() {
+    override fun getPlaceholderText(project: Project, lines: MutableList<String>): String? {
+        return "<" + lines.size + " internal calls>"
```

Comment:
1 call = 2 lines (not sure it's always true).

![image](https://user-images.githubusercontent.com/6079006/88716494-d1ae1d00-d127-11ea-8caa-5986a57584df.png)